app/views/service_requests/right_navigation/_short_interaction_modal.html.haml

```diff
@@ -0,0 +1,64 @@
+-# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+-# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#interaction-modal
+  .modal-dialog.interaction
+    .modal-content
+      = form_for @short_interaction, url: short_interactions_path, method: :post, remote: true do |f|
+        = f.hidden_field :identity_id, value: @identity.id
+        .modal-header
+          %button.close{ type: 'button', data: { dismiss: 'modal' } }
+            %span{ aria: { hidden: 'true' } } &times;
+          %h4.text-center#modal-title
+            = t(:proper)[:right_navigation][:short_interaction][:header]
+        .modal-body
+          #modal_errors
+          .form-group.row
+            = f.label :subject, t(:proper)[:right_navigation][:short_interaction][:subject], class: 'col-lg-4 control-label'
+            .col-lg-8
+              = f.select :subject, options_for_select(PermissibleValue.get_inverted_hash('interaction_subject')), :prompt =>t(:proper)[:right_navigation][:short_interaction][:subject_select], class: 'form-control'
+          .form-group.row
+            = f.label :interaction_type, t(:proper)[:right_navigation][:short_interaction][:type], class: 'col-lg-4 control-label'
+            .col-lg-8
+              = f.select :interaction_type, options_for_select(PermissibleValue.get_inverted_hash('interaction_type')), :prompt =>t(:proper)[:right_navigation][:short_interaction][:type_select], class: 'form-control'
+          .form-group.row
+            = f.label :duration_in_minutes, t(:proper)[:right_navigation][:short_interaction][:duration], class: 'col-lg-4 control-label'
+            .col-lg-8
+              = f.text_field :duration_in_minutes,  placeholder: t(:proper)[:right_navigation][:short_interaction][:duration_placeholder],  size: 65, class: 'form-control'
+          .form-group.row
+            = f.label :name, t(:proper)[:right_navigation][:short_interaction][:investigator_name], class: 'col-lg-4 control-label'
+            .col-lg-8
+              = f.text_field :name,  placeholder: t(:proper)[:right_navigation][:short_interaction][:investigator_name],  size: 65, class: 'form-control'
+          .form-group.row
+            = f.label :email, t(:proper)[:right_navigation][:short_interaction][:investigator_email], class: 'col-lg-4 control-label'
+            .col-lg-8
+              = f.text_field :email,  placeholder: t(:proper)[:right_navigation][:short_interaction][:investigator_email],  size: 65, class: 'form-control'
+          .form-group.row
+            = f.label :institution, t(:proper)[:right_navigation][:short_interaction][:institution], class: 'col-lg-4 control-label'
+            .col-lg-8
+              = f.select :institution, options_for_select(INSTITUTIONS), :prompt =>t(:proper)[:right_navigation][:short_interaction][:institution_select], class: 'form-control'
```

Comment:
See previous comment about the form selects. The same fix works here.

Reply:
They are years old code! Thanks for catching. I will update them and do new push